contracts/NFTStorefront.cdc

```diff
@@ -464,14 +599,16 @@ pub contract NFTStorefront {
         // Anyone can call, but at present it only benefits the account owner to do so.
         // Kind purchasers can however call it if they like.
         //
-        pub fun cleanup(listingResourceID: UInt64) {
+        access(contract) fun cleanup(listingResourceID: UInt64) {
```

Comment:
Just want to verify my understanding - this allows for destroying any listing (as long as it exists) and is no longer a public function (can only be called within the contract?

Reply:
Correct.